flaml/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.3.6"
```

Comment:
do not change version

Reply:
changing version number doesn't requires publishing to pypi.